CompetitionTemplate/src/competition/subsystems/vision/VisionStateMonitor.java

```diff
@@ -0,0 +1,40 @@
+package competition.subsystems.vision;
+
+import java.util.Observable;
+import java.util.Observer;
+
+import org.apache.log4j.Logger;
+
+import xbot.common.logic.Latch;
+import xbot.common.logic.Latch.EdgeType;
+
+public class VisionStateMonitor implements Observer {
```

Comment:
It's a neat concept to use Observer for these kinds of things, but in that case it would be nice to register them with some common system that will call update on them every loop instead?